site/content/docs/traces-open-telemetry/exporting-traces/dash0.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Exporting traces to Dash0 - Checkly Docs
+displayTitle: Exporting traces to Dash0
+navTitle: To Dash0
+description: "Export traces from Checkly to Dash0 for better observability."
+weight: 24
+menu:
+  platform:
+    parent: Export traces to 3rd party tools with Checkly - Checkly Docs
+beta: true
+aliases:
+  - "/docs/open-telemetry/exporting-traces/dash0/"
+---
+
+1. Create an API key in the **Settings** > **Auth Tokens** section.
+  ![Generate an auth token](/docs/images/otel/export-traces/dash0-token.png)
+2. Grab the right endpoint URL for your Dash0 organizsation. It should look like `https://ingress.eu-west-1.aws.dash0.com/v1/traces`.
```

Comment:
Where do people grab it? Could you add more details? If possible follow the example from Grafana which is the most straight-forward to follow,

```suggestion
2. Grab the right endpoint URL for your Dash0 organization. It should look like `https://ingress.eu-west-1.aws.dash0.com/v1/traces`.
```